### DISCUSSION TITLE
drawing tablet

### DISCUSSION THREAD

**Discussion by vishalkadam47** [kasmtech/KasmVNC#195] [Q&A]
Does kasmVNC have Pressure sensitive drawing tablet support

**Comment by mmcclaskey** [comment]
KasmVNC does not support a pressure sensitive input device. It is unlikely that we would implement this directly within KasmVNC. If there was enough demand for it, the feature would most likely be implemented within the Kasm Workspaces platform, out of band from KasmVNC.

**Comment by vishalkadam47** [comment]
I don't think there would be a demand for this, I have a plan for deploying KasmVNC for my team it would be helpful if there is a control for Wacom creative pen tablet.

**Comment by webysther** [comment]
I have installed the LSIO Krita and discovered that the stylus pressure don't work, make almost useless the krita, inkscape and gimp for drawing. Maybe sunshine/moonlight work for this, I don't know.

**Comment by webysther** [comment]
Yeap, sunshine have initial support for pen https://github.com/LizardByte/Sunshine/pull/1483 For loupedeck keyboard and other types of MIDI looks is working just fine, inside krita the pen is detected as a mouse and no pressure is enabled.

**Comment by webysther** [comment]
Parsec have added support at same time last year https://support.parsec.app/hc/en-us/articles/4423579900301-Using-Wacom-Tablets but still in beta
